### PR TITLE
Fix/ole emf header alternative

### PR DESCRIPTION
**Pull request by OssianEPPlus**
Before submitting this pull request I have...
- [X] read EPPlus Softwares [Contribution Guidlines](CONTRIBUTING.md)
- [X] ensured that the functionality I have added/changed is covered by new unit tests.
- [X] merged the target branch into the PR branch and resolved any merge conflicts
- [X] Run all the unit tests and ensured that they all are green (unless the purpose of the PR is to provide us with failing unit tests).

Added alternate headers and font handling.
Also added correct character spacing for emf files based on fonts

### CODE REVIEW

File: src/EPPlusTest/EmfTest.cs
Comment (swmal) [inline src/EPPlusTest/EmfTest.cs:1]: Commented code on line 64

File: src/EPPlus/Drawing/EMF/Records/EMR_EXTCREATEFONTINDIRECTW.cs
Comment (swmal) [inline src/EPPlus/Drawing/EMF/Records/EMR_EXTCREATEFONTINDIRECTW.cs:1]: Commented code on line 46-51

File: src/EPPlus/Drawing/EMF/Records/EMR_EXTTEXTOUTW.cs
Comment (swmal) [inline src/EPPlus/Drawing/EMF/Records/EMR_EXTTEXTOUTW.cs:1]: Commented code on line 68 and 69

File: src/EPPlus/Drawing/EMF/Records/EMR_HEADER.cs
Comment (swmal) [inline src/EPPlus/Drawing/EMF/Records/EMR_HEADER.cs:1]: Commented code on line 142-143

File: src/EPPlus/Drawing/EMF/Enums/MapModeEnum.cs
Comment (swmal) [inline src/EPPlus/Drawing/EMF/Enums/MapModeEnum.cs:1]: Commented code on lines 50-57

**Review by swmal — CHANGES_REQUESTED** [review]
Looks very good. I have added comments in this review where code has been disabled with comments.

File: src/EPPlus/Core/Worksheet/Fonts/GenericFontMetrics/GenericFontMetricsTextMeasurer.cs [hunk 1]
```diff
@@ -14,8 +14,7 @@ Date               Author                       Change
 using OfficeOpenXml.Interfaces.Drawing.Text;
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+using System.Drawing;
```
Comment (JanKallman) [inline src/EPPlus/Core/Worksheet/Fonts/GenericFontMetrics/GenericFontMetricsTextMeasurer.cs]: Should be removed?

Comment (OssianEPPlus) [reply src/EPPlus/Core/Worksheet/Fonts/GenericFontMetrics/GenericFontMetricsTextMeasurer.cs]: yep

File: src/EPPlus/Drawing/EMF/Enums/MapModeEnum.cs
Comment (OssianEPPlus) [reply src/EPPlus/Drawing/EMF/Enums/MapModeEnum.cs:1]: Fixed

File: src/EPPlus/Drawing/EMF/Records/EMR_EXTCREATEFONTINDIRECTW.cs
Comment (OssianEPPlus) [reply src/EPPlus/Drawing/EMF/Records/EMR_EXTCREATEFONTINDIRECTW.cs:1]: fixed

File: src/EPPlus/Drawing/EMF/Records/EMR_EXTTEXTOUTW.cs
Comment (OssianEPPlus) [reply src/EPPlus/Drawing/EMF/Records/EMR_EXTTEXTOUTW.cs:1]: fixed

File: src/EPPlus/Drawing/EMF/Records/EMR_HEADER.cs
Comment (OssianEPPlus) [reply src/EPPlus/Drawing/EMF/Records/EMR_HEADER.cs:1]: Fixed. Do note that we Might need the values again some day. If so, refer to this github comment to find them.

```
//Bounds = new byte[16] { 0x13, 0x00, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00, 0x4b, 0x00, 0x00, 0x00, 0x30, 0x00, 0x00, 0x00 };
//Frame = new byte[16] { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xBD, 0x08, 0x00, 0x00, 0x90, 0x06, 0x00, 0x00 };
```

File: src/EPPlusTest/EmfTest.cs
Comment (OssianEPPlus) [reply src/EPPlusTest/EmfTest.cs:1]: fixed

**Pull request merged**